src/NBitcoin/BIP9DeploymentsArray.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace NBitcoin
```

Comment:
If you make changes here can we look at moving all deployment code to Stratis.Bitcoin.Base.Deplyoments folder?